Dockerfile

```diff
@@ -6,8 +6,7 @@ LABEL homepage="https://nodesource.com"
 LABEL maintainer="NodeSource"
 
 RUN apt-get update && apt-get install -y g++ build-essential
-RUN npm install -g ncm-cli
-
-COPY entrypoint.sh /entrypoint.sh
+RUN npm install -g https://github.com/nodesource/ncm-cli.git#gate
```

Comment:
https://github.com/nodesource/ncm-cli.git#gate redirects to https://github.com/nodesource/ncm-cli.git
I'm not sure, but maintain the `#gate` have some differences?

Reply:
reverted this because this was for experimental usage